Makefile

```diff
@@ -34,8 +34,9 @@ fix-services: $(SERVICES_DIR)/*
 	sed -i '/^export abstract class.*/a protected abstract config: ClientConfig;' $^
 	sed -i '/^export abstract class.*/a protected abstract client: AxiosInstance;' $^
 	sed -i 's/public static/public/' $^
+	sed -i -r 's/\): CancelablePromise</overrideAccount?: string\): CancelablePromise</' $^
 	sed -i -r 's/CancelablePromise<(.*)>/Promise<Result<\1, ApiError>>/' $^
-	sed -i 's/return __request(ClientConfig,/return __request(this.client, this.config,/' $^
+	sed -i 's/return __request(ClientConfig,/return __request(overrideAccount, this.client, this.config,/' $^
 	# [ESM support] Update imports to point to .js file
```

Comment:
I think we should remove this altogether.
It's too flaky and untested.

Reply:
You mean sed? I agree if so